eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -54,13 +53,14 @@ const AttendanceCard: React.FC<Props> = ({
 
   const timeString = `${startTime} - ${endTime}`;
 
+
   return (
     <div className={classes.wrapper}>
       <ListItem className={classes.listHeader}>
         <ListItemText
           primary={
             <span>
-              {timeString} <b>({userBookings.length})</b>
+              {"13:00 - 14:00"} <b>({userBookings.length})</b>
```

Comment:
You can calculate `startTime` as lowest `startTime` of all intervals and `endTime` as highest `endTime`of all intervals. Please avoid hardcoded values and pass everything from storybook.